ChechpointStyles.css

```diff
@@ -0,0 +1,78 @@
+
+body { background-color: silver;
+
+}
+
+
+
+nav { display: block;
+     padding: 40px;
+     background-color: green;
+     
+}
+
+a {color: antiquewhite;
+   font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-weight: 700;
+}
+
+.float-left{ float: left;
+display: block;
+padding: 8px;
+border-right: 1px solid whitesmoke;
+text-align: center;
+}
+
+.float-right{ float: right;
+display: block;
+padding: 8px;
+border-right: 1px solid whitesmoke;
+}
+
+h1 { text-align: center;
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+text-decoration: underline;}
+
+#IMAGE { text-align: center;
+}
+
+h2{
+    color: sienna;
+    font-style: oblique;
+    
+}
+
+p { font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-weight: bolder;
+    
+}
+
+#Headline { text-decoration: underline;
+border: dashed black medium;
+color:seagreen;
+}
+
+.Head {
+background-color: crimson;
+font-weight:900;
+font-style: italic;
+font-family: 'Times New Roman', Times, serif;
```

Comment:
I see many `font-family` declarations, this is probably not very good. Usually, a page would have 1 or 2 font families used.

If you would like to use the same family across the website, you can define it in the `body` e.g.:

```css
body {
  font-family: 'Times New Roman', serif;
  font-weight: 500;
}
```